### PR TITLE
Add loggers in collector/modifier

### LINKED ISSUE #446: Entire codesniffer JS source file is written to logs for each accessibility collect
**Describe the bug**
When using the accessibility collector in a suite, the entire  codesniffer JS source file is written to logs for each accessibility collection. This happens only in the DEBUG logs level - however this should not happen even on this level - this JS file contains few thousands lines of code.

CC @plutasnyy 

* Version used: `AET 3.1.0`

**To Reproduce**
0. Make sure that the Logger on your AET environement (karaf) is set to DEBUG mode
1. Prepare a suite XML with a test using accessibility collector.
2. Run the suite
3. View the `worker.log` file - see the entry starting with
`<time> | DEBUG | com.cognifide.aet.jobs   | JavaScriptJobExecutor              51 | Executing JavaScript command: /*` followed by the whole source of [htmlcs.js file](https://github.com/Cognifide/aet/blob/master/core/jobs/src/main/resources/collectors/accessibility/htmlcs.js)

**Expected behavior**
Option 1 - When writing the JS code snippet to the log file, it's length should be limited to some reasonable value (e.g. first 100 characters). 
Option 2 -The Accessibility collector should not log the content of executed JS file at all

### PR DESCRIPTION
**Pull request by Asia95**
I made changes in loggers as suggested in #446 

## Description
- Change log level in [JavaScriptJobExecutor.executeJs](https://github.com/Cognifide/aet/blob/master/core/jobs/src/main/java/com/cognifide/aet/job/common/utils/javascript/JavaScriptJobExecutor.java#L50) to TRACE
- Add separate DEBUG level logs in every collector/modifier that calls [JavaScriptJobExecutor.executeJs](https://github.com/Cognifide/aet/blob/master/core/jobs/src/main/java/com/cognifide/aet/job/common/utils/javascript/JavaScriptJobExecutor.java#L50)

## Motivation and Context

Closes #446 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] I have reviewed (and updated if needed) the documentation regarding this change

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: core/jobs/src/main/java/com/cognifide/aet/job/common/utils/javascript/JavaScriptJobExecutor.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public JavaScriptJobResult execute(String jsSnippet, Object... elements)
 
   private JavaScriptJobResult executeJs(String jsSnippet, Object... elements) {
     if (LOGGER.isDebugEnabled()) {
-      LOGGER.debug("Executing JavaScript command: {} on page: {}", jsSnippet, currentUrl);
+      LOGGER.trace("Executing JavaScript command: {} on page: {}", jsSnippet, currentUrl);
```
Comment (plutasnyy) [inline core/jobs/src/main/java/com/cognifide/aet/job/common/utils/javascript/JavaScriptJobExecutor.java]: Thanks! 👍

File: core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/accessibility/AccessibilityCollector.java [hunk 2]
```diff
@@ -60,8 +64,10 @@
   @Override
   public CollectorStepResult collect() throws ProcessingException {
     String script = getScriptFromFile();
+    LOGGER.debug("Executing Accessibility Collector");
     final String html = jsExecutor.execute(DOCUMENT_OUTER_HTML_SCRIPT)
         .getExecutionResultAsString();
+    LOGGER.debug("Executing Accessibility Collector");
```
Comment (plutasnyy) [inline core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/accessibility/AccessibilityCollector.java]: Duplicate messages were on purpose?

File: core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/executejavascript/ExecuteJavaScriptModifier.java [hunk 3]
```diff
@@ -56,6 +56,7 @@ public CollectorStepResult collect() throws ProcessingException {
     CollectorStepResult result;
     String jsSnippet = getJsSnippet();
     try {
+      LOG.debug("Executing Java Script Modifier");
```
[the remark this answers is not in the document]
Comment (Asia95) [reply core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/executejavascript/ExecuteJavaScriptModifier.java]: fixed

File: core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/accessibility/AccessibilityCollector.java (hunk 2 above)
Comment (Asia95) [reply core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/accessibility/AccessibilityCollector.java]: fixed :)

**Pull request merged**